cerberus/aws_auth.py

```diff
@@ -65,6 +55,28 @@ def set_auth(self, role_arn=None, region=None, assume_role=True):
         else:
             self.region = region
 
+    def get_ecs_role_arn(self):
```

Comment:
Should there be tests added to [test_aws_auth](https://github.com/Nike-Inc/cerberus-python-client/blob/master/tests/test_aws_auth.py) for this new logic?